luvit-loader.lua

```diff
@@ -207,7 +207,7 @@ local function loader(dir, path, bundleOnly)
     if package.loaded[fullPath] then
       return package.loaded[fullPath]
     end
-    local module = assert(loadfile(fullPath))()
+    local module = assert(loadfile(fullPath))(fullPath)
```

Comment:
It allows usage of the [module](http://lua-users.org/wiki/AlternativeModuleDefinitions) function like so:

```lua
module(...)
```

`module` is deprecated/removed in Lua > 5.1, but Lua still passes the module name as a parameter to loaded files. In [Lua > 5.1, the filename is also passed as the second parameter to the loaded module](https://github.com/lua/lua/blob/5.2.x/src/loadlib.c#L400).